charts/aws-ebs-csi-driver/templates/snapshot-controller.yaml

```diff
@@ -23,40 +23,33 @@ spec:
       nodeSelector:
         kubernetes.io/os: linux
         {{- with .Values.nodeSelector }}
-{{ toYaml . | indent 8 }}
+          {{- toYaml . | nindent 8 }}
         {{- end }}
       priorityClassName: {{ .Values.priorityClassName | default "system-cluster-critical" }}
       {{- with .Values.affinity }}
-      affinity: {{ toYaml . | nindent 8 }}
+      affinity:
+        {{- toYaml . | nindent 8 }}
       {{- end }}
       tolerations:
-        {{- if .Values.tolerateAllTaints }}
-        - operator: Exists
-        {{- else }}
         - key: CriticalAddonsOnly
           operator: Exists
         - operator: Exists
           effect: NoExecute
           tolerationSeconds: 300
-        {{- end }}
         {{- with .Values.tolerations }}
-{{ toYaml . | indent 8 }}
+        {{- toYaml . | nindent 8 }}
```

Comment:
how can we ensure indenting stays clean as we continue editing these templates? is this just a vanilla yaml linter fixing it up?

Reply:
I changed this for readability.  Not with a linter or other auto process.  I haven't really found one that formats the helm templates in a very readable way.

Reply:
ok. seems i need to simply do a better job of reviewing then : p